src/runvm-osbuild

```diff
@@ -124,6 +124,8 @@ osbuild \
     --checkpoint build           \
     --checkpoint tree            \
     --checkpoint raw-image       \
+    --checkpoint raw-metal-image \
+    --checkpoint raw-meta4k-image \
```

Comment:
Based on the current version of the manifests I think you want `--checkpoint metal --checkpoint metal4k`. I also think maybe we should bump the `--cache-max-size 14GiB` to `20GiB`.